types/rbactypes.go

```diff
@@ -49,10 +49,6 @@ type Binding struct {
 	Roles             []string  `bson:"roles" json:"roles,omitempty"`
 }
 
-type BindingFilter struct {
```

Comment:
?

Reply:
?

Reply:
(it's no longer used since it has been incapsulated within [the lib](github.com/mia-platform/go-crud-service-client)